Taskfile.yaml

```diff
@@ -71,6 +71,15 @@ tasks:
       - .github/check-ci-files.sh charts/redpanda/ci
       - go vet ./...
 
+  ci:kube-lint-template:
```

Comment:
Could we instead run this via `go test`? Would love to add this as a second step to `TestTemplate`. Should be fine to run through `os/exec`. There's a JSON output if we want to parse it and that would allow us to manage the ignores in gocode as well.

Reply:
cool idea, will do.

Reply:
That step could be perform after `helm template`

https://github.com/redpanda-data/helm-charts/blob/6449ce3afcc6bb91607968accded1935ffab52e5/charts/redpanda/chart_test.go#L127-L140

Reply:
I think this PR missing a removal of `ci:kube-lint-template` task target

Reply:
No, that was intentional, i would like to run this manually but ok...  ill remove it